packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -128,6 +169,43 @@ export class AzureClient {
         return { container: fluidContainer, services };
     }
 
+    /**
+     * Get the list of versions for specific container
+     * @param id - Unique ID of the source container in Azure Fluid Relay.
+     * @param maxCount - Max number of versions to retreive,
+     * @returns Array of available versions
+     */
+     public async getContainerVersions(
+        id: string,
+        maxCount: number,
```

Comment:
we could make this optional and max to some small number.

Reply:
I think this seems like a good idea. Let's consult FRS team on the number here. I think they have some default in mind that they want to keep for now until the number is configurable.

Reply:
Based on conversation with FRS team they will not limit number of summaries they are going to keep. Later, in the next month or so, when they move from PragueGraph to a Git-based solution, that behavior should remain the same. I am keeping maxCount at 5. It's small enough and it lines up with our use case. Even if we end up extending this further and inserting pagination single "fetch" count may remain small. Anyways, we can always adjust this later without breaking API. I have ended up adding `AzureGetVersionsOptions`  (that contains `maxCount`), so we can extend these get options in future.